lab_1_classify_by_unigrams/main.py

```diff
@@ -8,35 +8,55 @@
 
 def tokenize(text: str) -> list[str] | None:
     """
-    Split a text into tokens.
-
-    Convert the tokens into lowercase, remove punctuation, digits and other symbols
-
-    Args:
-        text (str): A text
-
-    Returns:
-        list[str] | None: A list of lower-cased tokens without punctuation
-
-    In case of corrupt input arguments, None is returned
-    """
+Split a text into tokens.
+Convert the tokens into lowercase, remove punctuation, digits and other symbols
+Args:
+    text (str): A text
+Returns:
+    list[str] | None: A list of lower-cased tokens without punctuation
+In case of corrupt input arguments, None is returned
+"""
+    if not isinstance(text, str):
+        return None
+    low = text.lower()
+    symbols = []
+    for symbol in low:
```

Comment:
can you combine these 2 for loops?

Reply:
probably, i can register it immediately in return. with another loop, it gives an error. 

done